doc/source/language/features.rst

```diff
@@ -66,10 +66,9 @@ Structures are variables that contain more than one piece of information. For ex
     PRINT "The Mun's periapsis altitude is: " + MUN:PERIAPSIS.
     PRINT "The ship's surface velocity is: " + SHIP:VELOCITY:SURFACE.
 
-.. versionadded:: 0.19.0
-    As of kOS version 0.19.0, in fact ALL values a script can see are
-    now a kind of :struct:`Structure <structure>`, even basic primitive
-    types such as Boolean and Scalar.
+.. note::
+    .. versionadded:: 0.19.0
+        As of kOS version 0.19.0, in fact ALL values a script can see are now a kind of :struct:`Structure <structure>`, even basic primitive types such as Boolean and Scalar.
```

Comment:
That was probably an accident while doing the .. note thing. Switching it back.

Related: is there a convention for line breaks? There's some of everything.